models/threshold.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"github.com/prometheus/common/model"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/models/threshold"
+)
+
+
+type Thresholds []Threshold
+// Customize Thresholds error rates
+type Threshold struct {
```

Comment:
About naming, better "ThresholdHealth" for consistency wutg ServiceHealth, AppHealth, and others.